backend/core/urls.py

```
@@ -82,6 +82,8 @@
 router.register(r"findings", FindingViewSet, basename="findings")
 router.register(r"incidents", IncidentViewSet, basename="incidents")
 router.register(r"timeline-entries", TimelineEntryViewSet, basename="timeline-entries")
+router.register(r"task-templates", TaskTemplateViewSet, basename="task-templates")
```

Ruff (0.8.2)

`TaskTemplateViewSet` may be undefined, or defined from star imports

(F405)